tests/check-legal-aid/test/specs/child-benefits.js

```diff
@@ -154,7 +154,7 @@ module.exports = {
       );
     },
 
-  "Should also see fields if benefits=no but children=yes": !function(client) {
+  "Should also see fields if benefits=no but children=yes": function(client) {
```

Comment:
You're unskipping a test that is failing for a pretty involved reason... I think making this test pass will be out of scope here

Reply:
Yep, removed now